prepareTransaction/eosJS/holacontrato.js

```diff
@@ -0,0 +1,46 @@
+
+const eosUtil = require('./eos')
```

Comment:
@AngeloCG97  Have a loook at this documentation

https://developers.eos.io/manuals/eosjs/latest/transaction-examples